kustomize/overlays/eks/flyte/dependencies/alb_ingress/ingress_grpc.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
+    alb.ingress.kubernetes.io/backend-protocol-version: GRPC
+    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-east-2:590375264460:certificate/e2f04275-2dff-4118-a493-ed3ec8f41605
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
```

Comment:
`HTTP` is not supported for `GRPC` protocol

Error from AWS load balancer controller
```
InvalidLoadBalancerAction: Listener protocol 'HTTP' is not supported with a target group with the protocol-version 'GRPC'\n\tstatus code: 400
```

Reply:
Done